src/CoinEntry.h

```diff
@@ -17,22 +17,36 @@
 
 #include <string>
 #include <vector>
+#include <variant>
 #include <utility>
 
 namespace TW {
 
 typedef std::vector<std::pair<Data, Data>> HashPubkeyList;
 
+struct UTXOPrefix {
```

Comment:
Need to find a better name here... This is Bitcoin (+fork) specific, for 'legacy' non-segwit, Base58-encoded addresses.  `Base58Prefix`?